hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieLogFileReader.java

```diff
@@ -279,7 +277,7 @@ public void close() throws IOException {
   }
 
   @Override
-  /**
```

Comment:
Can we move `@Override` close to the method so that we do not need to change this `/**`. WDYT?

Reply:
done